backend/tests/api/announcement/index.js

```diff
@@ -0,0 +1,200 @@
+const request = require('supertest');
+const { jwt } = require("../../helpers/strapi");
+const userFactory = require("../../user/factory");
+
+describe("Announcement Test", () => {
+    let authenticatedUser;
+    let adminUser;
+
+    beforeAll(async () => {
+        authenticatedUser = await userFactory.createUser(strapi, "authenticated");
+        adminUser = await userFactory.createUser(strapi, "admin");
+    })
+
+    const constructAnnouncement = (id) => {
+        return {
+            data: {
+                "Title": "Test Announcement " + id,
+                "Poster": null,
+                "Description": "Announcement Description " + id,
+                // "HostingGroup": "Announcement HostingGroup" + id,
```

Comment:
Why is it commented out? Do we need to tag Hosting Group?

Reply:
I tried to test it, but it did not work because I need to set Hosting group beforehand. I can remove that line!